docs/reference/setup/install/docker.asciidoc

```diff
@@ -399,14 +399,53 @@ sudo sysctl -w vm.max_map_count=262144
 
 ====== Windows with https://docs.docker.com/docker-for-windows/wsl[Docker Desktop WSL 2 backend]
 
-The `vm.max_map_count` setting must be set in the docker-desktop container:
+The `vm.max_map_count` setting must be set in the "docker-desktop" WSL instance before the
+ElasticSearch container will properly start. There are several ways to do this, depending
+on your version of Windows and your version of WSL.
 
-[source,sh]
+If you are on Windows 10 before version 22H2, or if you are on Windows 10 version 22H2 using the
+built-in version of WSL, you must either manually set it every time you restart Docker before starting
+your ElasticSearch container, or (if you do not wish to do so on every restart) you must globally set
+every WSL2 instance to have the `vm.max_map_count` changed. This is because these versions of WSL
+do not properly process the /etc/sysctl.conf file.
+
+To manually set it every time you reboot, you must run the following commands in a command prompt
+or PowerShell window every time you restart Docker:
+
+[source]
 --------------------------------------------
-wsl -d docker-desktop
+wsl -d docker-desktop -u root
 sysctl -w vm.max_map_count=262144
+exit
```

Comment:
Do we need `exit` here?

Reply:
As much as I dislike mentioning this, many Windows users and administrators prefer simply closing the window they're working in to exiting -- because they usually can't figure out how to get back to the mode they had been in before. I view it as a potential place where they can learn about how to exit WSL, the same way many users [new to UNIX] need help understanding how to exit out of insert mode in vi.

However, I am not the one who's responsible for the tone of the documentation. That would be you and your organization. I'm simply stating my reason for it, and I won't be particularly sad if you decide it shouldn't be there.